Doc/library/socket.rst

```diff
@@ -1633,12 +1633,13 @@ sends traffic to the first one connected successfully. ::
        print('could not open socket')
        sys.exit(1)
    conn, addr = s.accept()
-   with conn:
-       print('Connected by', addr)
-       while True:
-           data = conn.recv(1024)
-           if not data: break
-           conn.send(data)
+   with s:
+        with conn:
```

Comment:
Since 3.1 and 2.7 you can put two context managers in one _with s, conn_ statement. Or you could extend the _with s_ bit to include the _s.accept_ call above.